relion/protocols/protocol_crop_resize.py

```diff
@@ -62,7 +64,7 @@ def _defineParams(self, form):
         form.addParam('rescaleSamplingRate', params.FloatParam,
                       default=1.0,
                       condition='doRescale',
-                      label='New sampling rate (Å/px)')
+                      label='New sampling rate (A/px)')
```

Comment:
Just for curiosity, what's the motivation of change? Is the Angstrom symbol not being displayed properly somewhere?

Reply:
No motivation, a ? symbol apears in my computer, not the A symbol. So I changed it. But it was ok. I will change it as it was